demo/Main.tscn

```diff
@@ -43,6 +43,9 @@ directional_shadow_max_distance = 20.0
 
 [node name="FPSController" parent="." instance=ExtResource( 4 )]
 
+[node name="Configuration" parent="FPSController" index="0"]
+refresh_rate = 0.0
+
```

Comment:
This looks like a bug in GDNative, this property is set to not be available in the editor.. alas can't do much harm.